### PR TITLE
Jenkinsfile: Build and archive release media for Morello

### PR DESCRIPTION
**Pull request by jrtc27**

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -167,8 +181,12 @@ def maybeArchiveArtifacts(params, String suffix) {
             // Archive disk image
             sh label: 'Compress kernel and images', script: """
 rm -fv *.img *.xz kernel*
```
Comment (bsdjhb) [inline Jenkinsfile:183]: Does this rm of *.img throw away memstick.img before you save it below?

Comment (jrtc27) [reply Jenkinsfile:183]: No, it's put in tarball/ (as the default output root for jenkins-cheribuild) and moved out below

File: Jenkinsfile [hunk 2]
```diff
@@ -195,10 +213,11 @@ rm -rf tarball artifacts-*
 chmod +w *.xz
 mkdir -p "artifacts-${suffix}"
 mv -v *.xz "artifacts-${suffix}"
+[ ! -d ftp ] || tar -cJvf "artifacts-${suffix}/cheribsd-ftp-${suffix}.tar.xz" ftp
```
Comment (brooksdavis) [inline Jenkinsfile:216]: Is this the already compressed distribution tarballs?

Comment (jrtc27) [reply Jenkinsfile:216]: Yeah. I could skip the compression here, but want to at least tar it up, and .tar.xz keeps things consistent.

**Review by bsdjhb — APPROVED** [review]

**Review by brooksdavis — APPROVED** [review]

**Pull request merged**